src/store/Store.ts

```diff
@@ -30,13 +30,21 @@ export interface ServiceGraphDataState {
   graphData: any;
 }
 
+export interface ServiceGraphState {
+  dataState: ServiceGraphDataState;
```

Comment:
`dataState` is redundant in a Redux store everything is dataState. Let's be more precise here. The protocol we have been following here is just to name it the same as the type, so `serviceGraphDataState`